ClassHierarchyAndCollections/ClassHierarchyAndCollectionsTests/AssociationTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ClassHierarchyAndCollections;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace ClassHierarchyAndCollectionsTests
+{
+    [TestClass]
+    public class AssociationTests
+    {
+        [TestMethod]
+        public void Can_Create_Association()
+        {
+            var association = new Association("Association Name", "555-555-5555", DateTime.Now, 7.00m, DateTime.Now.AddMonths(1));
+            Assert.IsInstanceOfType(association, typeof(Association));
+        }
+
+        [TestMethod]
+        public void Can_Read_Write_Properties()
```

Comment:
Going forward, please use a different method for each property. That way, if a test fails, you know immediately which property is not working.